scripts/infra

```diff
@@ -53,6 +53,9 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
 
         aws s3 cp "s3://${PFB_SETTINGS_BUCKET}/terraform/terraform.tfvars" "${PFB_SETTINGS_BUCKET}.tfvars"
 
+        # Workaround Git wanting everyone to be the same user
+        CURRENT_UID=$(id -u):$(id -g)
```

Comment:
From [this comment](https://github.com/azavea/pfb-network-connectivity/pull/896#issuecomment-1181783109):
> It was echo locator where I saw the permission/safe directory issue before. In that project I worked around it via adding:
git config --global --add safe.directory /usr/local/src to the infra script itself.

Do we want to do that and back out this workaround?  Or are we fine with the CURRENT_USER approach?

Reply:
Backing it out and switching to git config.